src/wrappers.jl

```diff
@@ -318,6 +318,11 @@ convert(::Type{Point}, ::PointTrait, geom) = Point(geom)
 convert(::Type{Point}, ::PointTrait, geom::Point) = geom
 extent(trait::PointTrait, geom::Point) = extent(trait, parent(geom)) 
 
+coordtype(::Point{<:Any,<:Any,<:AbstractArray{T}}) where T = T 
+coordtype(::Point{<:Any,<:Any,<:NTuple{<:Any,T}}) where T = T 
+coordtype(::Point{<:Any,<:Any,<:NamedTuple{<:Any,<:NTuple{<:Any,T}}}) where T = T 
```

Comment:
Should we remove the `Point` from here, so that the dispatch with `parent(p)` for `Point` feeds into these?

Reply:
I think that will hit the fallback method in fallbacks.jl. It just get the X coord type